crypto/aes/asm/bsaes-armv7.pl

```diff
@@ -49,10 +48,7 @@
 #						<appro@openssl.org>
 
 # April-August 2013
-#
-# Add CBC, CTR and XTS subroutines, adapt for kernel use.
-#
-#					<ard.biesheuvel@linaro.org>
+# Add CBC, CTR and XTS subroutines, adapt for kernel use courtesy of Ard.
```

Comment:
I'm not sure, but I feel that a punctuation mark is missing prior "courtesy". Maybe make make it a separate sentence, or would comma do?

Reply:
yeah, i tweaked it.
pushing a new commit